.travis.yml

```diff
@@ -0,0 +1,22 @@
+dist: xenial
+
+language: node_js
+
+node_js:
+  - node
+
+sudo: false
+
+addons:
+  - chrome: stable
+  
+install:
+  - npm install web
+
+script:
+  - npm ci --prefix=./web/
+  - npm run --prefix=./web/ lint
+  - npm ci --prefix=./web/
+  - npm run --prefix=./web/ test -- --watch=false
+  - npm ci --prefix=./web/
+  - npm run --prefix=./web/ e2e
```

Comment:
Can we do a `cd  web` at the beginning and avoid passing `--prefix=./web/`? If this are run in a single shell session we can run the `npm ci` only at the beginning.